File: 프로그래머스/2/64065. 튜플/튜플.py
def solution(s):
    answer = []
    nested_lists = [list(map(int, inner.strip('{}').split(','))) for inner in s.strip('{}').split('},{')]
    nested_lists.sort(key = lambda x:len(x))
    for i in nested_lists:
        for j in i:
            if j in answer:
                continue
            else:
                answer.append(j)
    return answer

File: 프로그래머스/2/64065. 튜플/튜플.js
function solution(s) {
    let answer = [];
    let str = s.replace(/{/g, "[").replace(/}/g,"]");
    let arr = JSON.parse(str);
    arr.sort((a,b) =>{
        return a.length - b.length;
    });
    for(let i = 0; i< arr.length; i++){
        for(let j=0; j<arr[i].length; j++){
            if(answer.includes(arr[i][j])){
                continue;
            }
            else{
                answer.push(arr[i][j]);
            }
        }
    }
    console.log(arr);
    return answer;
}